src/nvim/move.PVS-Studio.cfg

```diff
@@ -0,0 +1,10 @@
+
+source-file=/home/zyx/a.a/Proj/c/neovim/src/nvim/move.c
```

Comment:
Especially how it appeared *alone*. I would understand if there was one cfg per .c file, could not possibly let that sneak in. But cfg for only one source file…